android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -317,6 +324,20 @@ protected void doInBackgroundGuarded(Void... params) {
         }
         selection.replace(selection.length() - 1, selection.length(), ")");
       }
+
+      if (mFromTime > 0) {
+        selection.append(" AND " + Images.Media.DATE_TAKEN + " > ?");
+        selectionArgs.add(mFromTime + "");
+      }
+      long toTime = mToTime;
+      if (!TextUtils.isEmpty(mAfter)) {
+        toTime = Long.parseLong(mAfter);
+      }
+      if (toTime > 0) {
+        selection.append(" AND " + SELECTION_DATE_TAKEN);
+        selectionArgs.add(toTime + "");
+      }
```

Comment:
With the merging of #130, this can now be simplified to:

```java
      if (mFromTime > 0) {
        selection.append(" AND " + Images.Media.DATE_TAKEN + " > ?");
        selectionArgs.add(mFromTime + "");
      }
      if (mToTime > 0) {
        selection.append(" AND " + Images.Media.DATE_TAKEN + " <= ?");
        selectionArgs.add(mToTime + "");
      }
```
Which also has the side benefit of being able to now paginate through the results when they're filtered by time. Free wins are fun!